src/settings/settings.js

```diff
@@ -0,0 +1,11 @@
+import "./settings.scss";
+import { setDataInStorage } from "../utils/localStorage/localStorage.js";
+
+const SAVE_BUTTON = document.querySelector(".section__saveButton");
+const INPUT = document.querySelector(".section__input--playerName");
+
+const clickToSavePlayerName = () => {
```

Comment:
Shouldn't there be also a function, that sets input default value based on the player name in localStorage if it exists?

Reply:
And WDYT about calling it `onButtonClick`?